addons/medical_feedback/RscInGameUI.hpp

```diff
@@ -0,0 +1,13 @@
+class RscPictureKeepAspect;
+class RscInGameUI {
+    class RscStanceInfo {
+        controls[] += {QGVAR(bloodVolumeIndicator)};
+        class GVAR(bloodVolumeIndicator): RscPictureKeepAspect {
+            onLoad = QUOTE(with uiNamespace do {GVAR(bloodVolumeIndicator) = _this select 0});
```

Comment:
What's with you and `with` lol

Reply:
It's sound pretty and i don't feel lonely :D

Reply:
Please change it to `getVariable` though.

Reply:
@commy2 it would be `setVariable` and would require ARR_N, I think that is why he did it this way

Reply:
Yes, but I don't like mharis' way.

Reply:
with hurts performance (tiny bit) and hurts readability as its really uncommon in the codebase.
We've always done it with the ARR macros, I don't see a reason to change that.

Reply:
Restored pirate ARRGHHH! macro :D